bitrise.yml

```diff
@@ -317,6 +317,20 @@ workflows:
                   CACHE_ARCHIVE_URL="https://storage.googleapis.com/cache_pull_test_bucket/macos.tar.gz"
                 fi
                 envman add --key "CACHE_ARCHIVE_URL" --value $CACHE_ARCHIVE_URL
+      - script:
+          title: Use the archives stack id
```

Comment:
We are downloading pre-made caches from GCP and those caches were made with on old stack ids. This e2e test will fail unless the stack ids match. 

On the long run we should rethink how this test works and probably do something better but for now this seemed like the quickest solution.

Reply:
This PR contains a fix (setting the empty stack ID, so it is not checked), should merge that first https://github.com/bitrise-steplib/steps-cache-pull/pull/31

Reply:
Cool, I am happy with that.